config.yml

```diff
@@ -0,0 +1,12 @@
+version: 1
```

Comment:
What's this config?

Reply:
dependabot ignore. When you run tests, Stencil re-installs old Jest. Dependabot tries to re-install newer Jest. This reduces diff noise.